### ISSUE TITLE
logits

### ISSUE THREAD

**Issue by flipside17** [saharan/HGSL#7]
excuse me,I wonder what logits means in the trasin.py.Does it mean the representation of each node?

**Comment by saharan** [comment]
hey wrong repository to post!
there are no python files here.
